text/0984-drop-safari-less-than-14.md

```diff
@@ -0,0 +1,318 @@
+---
+stage: accepted
+start-date: 2023-11-02T15:40:00.000Z 
+release-date: # In format YYYY-MM-DDT00:00:00.000Z
+release-versions:
+teams: # delete teams that aren't relevant
+  - cli
+  - data
+  - framework
+  - learning
+  - steering
+  - typescript
+prs:
+  accepted: https://github.com/emberjs/rfcs/pull/984
+project-link:
+suite: 
+---
+
+
+<!--- 
+Directions for above: 
+
+stage: Leave as is
+start-date: Fill in with today's date, 2032-12-01T00:00:00.000Z
+release-date: Leave as is
+release-versions: Leave as is
+teams: Include only the [team(s)](README.md#relevant-teams) for which this RFC applies
+prs:
+  accepted: Fill this in with the URL for the Proposal RFC PR
+project-link: Leave as is
+suite: Leave as is
+-->
+
+# Treat Safari as an Evergreen Browser
+
+## Summary
+
+Safari's release cadence has increased as well as relevant-device compatibilty. This RFC proposes an amendment to the [browser support](https://emberjs.com/browser-support/) policy to treat Safari the same as Chrome and FireFox for Desktop and Mobile devices.
+
+## Motivation
+
+Private class fields transpilation has a big impact on the kind of code emitted. We want to transpile as little as possible -- ideally nothing, and let app users decide if they need to transpile further.
+
+## Detailed design
+
+Ember v6 will not support Safari < v14.1
+
+
+## How we teach this
+
+### A new browser support policy
+
+Diff:
+- remove non-evergreen section
+- place Safari under Evergreen Desktop and Evergreen Mobile
+- actual numbers subject to change as time passes, this is an example
+
+```diff
+{{page-title "Browser Support"}}
+<div class="container layout">
+  <section aria-labelledby="section-browser-support-policy">
+    <h1 id="section-browser-support-policy">Ember.js Browser Support Policy</h1>
+
+
++    <h2>Ember 6.0.0</h2>
++
++    <p>
++      In Ember 6.0.0, the framework supports the following major browsers:
++    </p>
++
++    <ul class="list-unstyled layout my-3">
++      <EsCard class="lg:col-2">
++        <div class="text-center text-md">Desktop</div>
++        <ul>
++          <li>Google Chrome >= 103</li>
++          <li>Mozilla Firefox >= 102</li>
++          <li>Microsoft Edge >= 110</li>
++          <li>Safari >= 16.4</li>
```

Comment:
actual numbers available here: https://browsersl.ist/#q=%3E+0.25%25+and+not+dead